statemachine/src/test/java/com/amazonaws/stepfunctions/cloudformation/statemachine/DeleteHandlerTest.java

```diff
@@ -149,7 +149,9 @@ public void testThrowsStateMachineDoesNotExist_whenStateMachineDoesNotExist() {
         final ProgressEvent<ResourceModel, CallbackContext> response
                 = handler.handleRequest(proxy, request, null, logger);
 
-        assertThat(response.getStatus()).isEqualTo(OperationStatus.SUCCESS);
+        assertThat(response.getStatus()).isEqualTo(OperationStatus.FAILED);
+        assertThat(response.getErrorCode()).isEqualTo(HandlerErrorCode.NotFound);
+        assertThat(response.getMessage()).contains(Constants.STATE_MACHINE_DOES_NOT_EXIST_ERROR_MESSAGE);
```

Comment:
nit: any reason we can't construct the expected response rather than verifying some of the attributes (multiple instances in this PR)

Reply:
Cleaned up test asserts to do object level assertions.